backend/prompt_studio/prompt_studio_core/prompt_studio_helper.py

```diff
@@ -580,6 +595,20 @@ def _fetch_response(
         Returns:
             Any: Output from LLM
         """
+
+        # Fetch the ProfileManager instance using the profile_manager_id if provided
+        if profile_manager_id:
+            try:
+                profile_manager = ProfileManager.objects.get(
```

Comment:
will be better if this function written in some helper class and call it from here.
except ProfileManager.DoesNotExist may return None from the heleper function, can be handled from here.

Reply:
Sure